jsapp/js/components/modalForms/translationSettings.es6

```diff
@@ -354,17 +373,15 @@ export class TranslationSettings extends React.Component {
               </React.Fragment>
             );
           })}
-          {!this.state.showAddLanguageForm &&
```

Comment:
This should stay as it was - now when you click 'Add language' and display the LanguageForm, the button does nothing

Reply:
Again, probably from old tests. Thanks for double checking it all!